tuna/solver_res.py

```diff
@@ -30,7 +30,8 @@
 
 LOGGER = setup_logger('solver_res')
 RES_FILENAME = 'solverres.csv'
-RES_FILE = open(RES_FILENAME, 'w')
+# pylint: disable=consider-using-with ; @alex tough making sure a global resource is closed?
+RES_FILE = open(RES_FILENAME, 'w') # pylint: disable=unspecified-encoding)
```

Comment:
good idea opening a resource as global?

Reply:
probably not, does this need to be global? can we minimize the scope?